docs/devGuide/maintainerGuide.md

```diff
@@ -333,6 +355,6 @@ test: Add diff printing for easier debugging (#632)
 
 f. Click "Publish release".
 
-11. Finally, announce the new release on our Slack channel (replace `A.B.C` with the new version). Congrats!
+12. Finally, announce the new release on our Slack channel (replace `A.B.C` with the new version). Congrats!
```

Comment:
If this is a Markdown list, we can use `1.` for every time, which Markdown will convert into a proper numbered list. That way, easier to insert items into the list in future.

Reply:
Unfortunately, because we have a bunch of other stuff in between each item, the numbering will reset for each point if we do that.